sklearnex/ensemble/_forest.py

```diff
@@ -78,7 +78,7 @@ def _onedal_fit(self, X, y, sample_weight=None, queue=None):
         )
 
         if sample_weight is not None:
-            sample_weight = self.check_sample_weight(sample_weight, X)
+            sample_weight = self._check_sample_weight(sample_weight, X)
```

Comment:
this was necessary to avoid the callable method discovery introduced in the new test coverage, this is shown in the changes in _utils.py. it is not a method of the sklearn estimator, but does take x as an input. it is used privately and should be made private.